js/blocks/VolumeBlocks.js

```diff
@@ -432,6 +432,9 @@ function setupVolumeBlocks(activity) {
                 if (typeof args[0] !== "number") {
                     activity.errorMsg(NANERRORMSG, blk);
                 } else {
+                    if (args[0] === 100 || args[0] === -100) {
+                        activity.errorMsg(_("Warning: Audio is currently going through a single channel."),blk);
```

Comment:
please add a space after the comma, before blk

Reply:
Added it in the new commit.